tools/txg/TXG/Types.hs

```diff
@@ -213,7 +226,11 @@ defaultArgs = Args
   , nodeVersion     = v
   , logHandleConfig = U.StdOut
   , batchSize       = BatchSize 1
-  , verbose         = Verbose False}
+  , verbose         = Verbose False
+  , gasLimit = 5000
```

Comment:
shouldn't these be set to constants https://github.com/kadena-io/chainweb-node/pull/794/files#diff-6364e1a9cc89ff20ebce9850b70ba28eR170